fullscreen.bs

```diff
@@ -68,13 +71,14 @@ is an <a>ordered set</a> of (<a>string</a>, <a>element</a>) <a>tuples</a>. It is
 
  <li><p>Set <var>element</var>'s <a>fullscreen flag</a>.
 
- <li><p><a for="top layer">Add</a> <var>element</var> to <var>element</var>'s <a>node document</a>'s
- <a>top layer</a>.
+ <li><p><a>Request removal from the top layer</a> given <var>element</var>.
+
+ <li><a>Add to the top layer</a> given <var>element</var>.
```

Comment:
Should this be using immediate removal rather than requesting removal? Transitions wouldn't work here since we're adding it immediately back to the top layer.

Reply:
It doesn't really matter which we use; they both produce identical behavior here.

Reply:
Reading the spec, it seems like they would but I don't see a reason to use the more complex one, when we can just use the simple one here.

Reply:
Yeah, seems clearer too.

Reply:
OK, let's go with immediate removal.